src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -139,6 +164,65 @@ public void merge_twoRepoConfigs_success() throws ParseException, IOException {
         Assert.assertEquals(REPO_LEVEL_GLOB_LIST, actualConfigs.get(0).getIgnoreGlobList());
     }
 
+    @Test
+    public void merge_emptyLocation_success() throws ParseException, IOException {
+        FIRST_AUTHOR.setIgnoreGlobList(FIRST_AUTHOR_GLOB_LIST);
+        SECOND_AUTHOR.setIgnoreGlobList(REPO_LEVEL_GLOB_LIST);
+
+        List<Author> expectedAuthorsBeta = new ArrayList<>();
+        expectedAuthorsBeta.add(FIRST_AUTHOR);
```

Comment:
*Can improve code quality using technique: use nouns for things and verbs for actions*
`https://se-edu.github.io/se-book/codeQuality/`

`expectedAuthorsBeta` -> `expectedBetaAuthors`